### PR TITLE
bug 1326074  - Make destrucive buttons more visible

### PR DESCRIPTION
**Pull request by thomasjosephgreco**
Added code for mozilla bug [1326074](https://bugzilla.mozilla.org/show_bug.cgi?id=1326074).

### CODE REVIEW

**Comment by thomasjosephgreco** [comment]
@mozbhearsum I created this PR for the issue we spoke about earlier. I ran the tests and everything passed but let me know if I missed anything 😄

File: ui/app/templates/releases.html [hunk 1]
```diff
@@ -72,8 +72,8 @@ <h3 class="panel-title">
       <div style="float:right">
         <i ng-if="release_name && $first && (currentPage == 1)">Current</i>
         <button ng-if="release_name && ((currentPage != 1) || !$first )" class="btn btn-default btn-xs" ng-click="openRevertModal(release)">Revert to this</button>
-        <button ng-show="!rule_id" class="btn btn-default btn-xs" ng-click="openNewScheduledReleaseChangeModal(release)">Schedule an Update</button>
-        <button ng-show="!rule_id" class="btn btn-default btn-xs" ng-click="openNewScheduledDeleteModal(release)">Schedule for Deletion</button>
+        <button ng-show="!rule_id" class="btn btn-default btn-xs" id="update-btn" ng-click="openNewScheduledReleaseChangeModal(release)">Schedule an Update</button>
+        <button ng-show="!rule_id" class="btn btn-default btn-xs" id="delete-btn" ng-click="openNewScheduledDeleteModal(release)">Schedule for Deletion</button>
```
Comment (mozbhearsum) [inline ui/app/templates/releases.html]: These colours are a little bit stark compared to the others. How about using the warning and danger classes that come with Bootstrap (http://getbootstrap.com/css/#buttons)? Those are a little dimmer, and would mean we don't have our own css to maintain either.

Comment (thomasjosephgreco) [inline ui/app/templates/releases.html]: 👍

File: ui/app/templates/releases.html [hunk 2]
```diff
@@ -72,8 +72,8 @@ <h3 class="panel-title">
       <div style="float:right">
         <i ng-if="release_name && $first && (currentPage == 1)">Current</i>
         <button ng-if="release_name && ((currentPage != 1) || !$first )" class="btn btn-default btn-xs" ng-click="openRevertModal(release)">Revert to this</button>
-        <button ng-show="!rule_id" class="btn btn-default btn-xs" ng-click="openNewScheduledReleaseChangeModal(release)">Schedule an Update</button>
-        <button ng-show="!rule_id" class="btn btn-default btn-xs" ng-click="openNewScheduledDeleteModal(release)">Schedule for Deletion</button>
+        <button ng-show="!rule_id" class="btn btn-warning btn-xs" id="update-btn" ng-click="openNewScheduledReleaseChangeModal(release)">Schedule an Update</button>
```
Comment (mozbhearsum) [inline ui/app/templates/releases.html]: Just remove these now-unnecessary ids and we're good to go!

**Pull request merged**